actionpack/lib/action_controller/metal/live.rb

```diff
@@ -343,15 +343,18 @@ def response_body=(body)
     #      end
     #    end
     def send_stream(filename:, disposition: "attachment", type: nil)
-      response.headers["Content-Type"] =
+      payload = { filename: filename, disposition: disposition, type: type }
+      ActiveSupport::Notifications.instrument("send_stream.action_controller", payload) do
+        response.headers["Content-Type"] =
         (type.is_a?(Symbol) ? Mime[type].to_s : type) ||
         Mime::Type.lookup_by_extension(File.extname(filename).downcase.delete("."))&.to_s ||
         "application/octet-stream"
```

Comment:
I think these lines should be indented along with the rest of the block:

```suggestion
          (type.is_a?(Symbol) ? Mime[type].to_s : type) ||
          Mime::Type.lookup_by_extension(File.extname(filename).downcase.delete("."))&.to_s ||
          "application/octet-stream"
```